packages/rocket-webhook-core/src/response/webhook-response-builder.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Headers,
+  WebhookHandlerFileReturnType,
+  WebhookHandlerJsonReturnType,
+  WebhookHandlerReturnType,
+  WebhookHandlerTextReturnType,
+  WebhookResponseType,
+} from '@boostercloud/rocket-webhook-types'
+import { WebhookFileResponse } from './responder/webhook-file-response'
+import { WebhookJsonResponse } from './responder/webhook-json-response'
+import { WebhookTextResponse } from './responder/webhook-text-response'
+import { WebhookEmptyResponse } from './responder/webhook-empty-response'
+
+export interface WebhookResponseClass {
+  getBody(response?: WebhookHandlerReturnType): unknown
+  getHeaders(response?: WebhookHandlerReturnType): Headers
+}
+
+export class WebhookResponseBuilder {
+  public static DEFAULT_HEADERS: Headers = {
+    'Access-Control-Allow-Origin': '*',
+  }
+
+  private webhookBaseResponse: WebhookResponseClass
+
+  constructor(readonly response?: WebhookHandlerReturnType) {
+    if (!response) {
+      this.webhookBaseResponse = new WebhookEmptyResponse()
+      return
+    }
+    if (this.isFile(response)) {
+      this.webhookBaseResponse = new WebhookFileResponse()
+      return
+    }
+    if (this.isJson(response)) {
+      this.webhookBaseResponse = new WebhookJsonResponse()
+      return
+    }
+    if (this.isText(response)) {
+      this.webhookBaseResponse = new WebhookTextResponse()
+      return
+    }
```

Comment:
I'm almost sure you don't need those `is...` functions, because you are not doing anything with the response and all the response types already have the `responseType` field.

So you can replace this with a `switch(response.responseType) { case WebhookResponseType.text: .... }` and avoid having those functions